### PR TITLE
fix: when searching and then closing search then title Threads or Discussions is not aligned in center

### LINKED ISSUE #5464: Title not aligned in center
### Description:

In ThreadMessagesVIew and DiscussionView, when searching and then closing search then title `Threads` or `Discussions` is not aligned in center

### Environment Information:

- Rocket.Chat Server Version: 6.6.0
- Rocket.Chat App Version: 4.44.0+
- Device Name: Realme 9
- OS Version: Android

### Steps to reproduce:

1. Go to Threads or Discussions
2. Click on Search
3. Click on close icon

### Expected behavior:

The title must align in center

### Actual behavior:

The title does not align in center

### Additional context:

https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/65113071/59bc4ad3-23d0-425a-8cdf-2e6b9321c506

### PR DESCRIPTION
**Pull request by SanjaySargam**
## Proposed changes
In ThreadMessagesVIew and DiscussionView, when searching and then closing search then title `Threads` or `Discussions` is not aligned in center

## Issue(s)	
Fixes #5464

## How to test or reproduce

1. Go to Threads or Discussions
2. Click on Search
3. Click on close icon 

## Screenshots

https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/65113071/f651d970-be76-4580-824e-03021838057d

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by GleidsonDaniel** [comment]
Have you checked if any behavior changes on iOS?

**Comment by SanjaySargam** [comment]
I don't have an IOS device how can I check ?

File: app/views/DiscussionsView/index.tsx [hunk 1]
```diff
@@ -124,6 +124,7 @@ const DiscussionsView = ({ navigation, route }: IDiscussionsViewProps): React.Re
 		options = {
 			headerTitleAlign: 'center',
 			headerTitle: I18n.t('Discussions'),
+			headerTitleContainerStyle: {  },
```
Comment (GleidsonDaniel) [inline app/views/DiscussionsView/index.tsx:127]:
```suggestion
			headerTitleContainerStyle: {},
```

File: app/views/ThreadMessagesView/index.tsx [hunk 2]
```diff
@@ -131,6 +131,7 @@ class ThreadMessagesView extends React.Component<IThreadMessagesViewProps, IThre
 		const options: StackNavigationOptions = {
 			headerTitleAlign: 'center',
 			headerTitle: I18n.t('Threads'),
+			headerTitleContainerStyle: { },
```
Comment (GleidsonDaniel) [inline app/views/ThreadMessagesView/index.tsx:134]:
```suggestion
			headerTitleContainerStyle: {},
```

**Review by GleidsonDaniel — APPROVED** [review]

**Pull request merged**